example/gardener-local/gardenlet/values-local-ha.yaml

```diff
@@ -0,0 +1,7 @@
+global:
```

Comment:
Nit: Let's rename this file to `values-kind-ha.yaml` (then it's even more clear to which kind cluster this configuration belongs).